.drone.yml

```diff
@@ -1,4 +1,47 @@
 kind: pipeline
+type: docker
+name: tests
+
+steps:
+  - name: gplay
+    image: nextcloudci/android:android-49
+    privileged: true
```

Comment:
I think you can remove this line AFAIR it is only needed for the emulator

Reply:
Confirmed on `testDroneFix` branch, I picked this PR without the priviliged line into mine to see if tests actually pass (they passed 🎉 ).